cve_bin_tool/cli.py

```diff
@@ -328,19 +328,15 @@ def main(argv=None, outfile=sys.stdout):
         choices=["debug", "info", "warning", "error", "critical"],
     )
     output_group.add_argument(
-        "-o",
-        "--output",
-        action="store",
-        choices=["csv", "json", "console"],
-        default="console",
-        help="update output format (default: console)",
+        "-o", "--output", action="store", default=None, help="provide output filename",
     )
     output_group.add_argument(
         "-f",
-        "--filename",
+        "--format",
         action="store",
-        default=None,
-        help="provide output file name",
+        choices=["csv", "json", "console"],
+        default="console",
```

Comment:
In practice, console and csv are the same right now.  Do you think we should just remove the console format until there's something different about it?

Reply:
Well unfortunately they are different at this point in time due to previous work, console does not write to file but csv/json always does. I can change this so output goes to stdout unless `--output-file` is given.

Reply:
That works, but maybe out of scope for this change and I should file it as a separate issue.